crates/torii/graphql/src/query/mod.rs

```diff
@@ -95,6 +95,16 @@ fn parse_nested_type(
     TypeData::Nested((TypeRef::named(target_type), nested_mapping))
 }
 
+fn remove_trailing_zeros(value: Value) -> Value {
```

Comment:
I think this should be `remove_leading_zeros`?

Reply:
Oops, updated!